pkg/controller/controller.go

```diff
@@ -98,8 +98,6 @@ func (c *Controller) CreateCluster(cluster model.Cluster, assets model.Assets) e
 	if cluster.Labels == nil {
 		cluster.Labels = model.Labels{}
 	}
-	// Set default cluster labels.
-	cluster.Labels[constants.ClusterNameLabelKey] = cluster.Name
 
```

Comment:
It needs to filter on the current cluster AND `stack-type=computepool`. It would seem as though the cluster-name tag IS set correctly on ALL nodes (inherited I guess).